.changeset/unlucky-oranges-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-input": minor
+---
+
+[Added] - Make the `value` property required and default to an empty string
```

Comment:
If it's required, do we need a default?

Reply:
<img width="1060" alt="Screenshot 2024-01-25 at 17 25 29" src="https://github.com/justeattakeaway/pie/assets/16766185/27925601-db3c-4419-b625-a2dbfc6a4a4e">